README.md

```diff
@@ -96,44 +104,44 @@ The original block reward schedule, with one-minute block targets and four-hour
 
   - [OSX Build Notes](doc/build-osx.md)
   - [Unix Build Notes](doc/build-unix.md)
-  - [Windows Build Notes](doc/build-windows.md)
+  - [Windows Build Notes](doc/build-msw.md)
 
 ### Such ports
-RPC 22555
-P2P 22556
+- RPC 22555
+- P2P 22556
 
-![](https://dogesay.com/3f5a00acf72df93528b6bb7cd0a4fd0c.jpeg)
+## Development tips and tricks 🖥️📝
 
-## Development tips and tricks
+**Compiling for debugging**
 
-**compiling for debugging**
+Run `configure` with the `--enable-debug` option, then `make`. Or run configure with
```

Comment:
please rebase your commit off #1766 / https://github.com/leuqarte/dogecoin/tree/1.14.3-dev to not have the same changes twice.